### ISSUE TITLE
Broken models in Electrical.Analog.Ideal due to undeclared variable

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#160]
**Reported by dietmarw on 10 Apr 2009 16:37 UTC**
The enhancement with the losses introduced some errors:
```
Checking model Modelica.Electrical.Analog.Ideal.ControlledIdealOpeningSwitch:
 
Error: Use of undeclared variable v
In class Modelica.Electrical.Analog.Ideal.ControlledIdealOpeningSwitch.
  In file: /Modelica/Electrical/Analog/Ideal.mo
 
Error: Use of undeclared variable i
In class Modelica.Electrical.Analog.Ideal.ControlledIdealOpeningSwitch.
  In file: /Modelica/Electrical/Analog/Ideal.mo
Errors found in: LossPower = v*i
  File: /Modelica/Electrical/Analog/Ideal.mo, line 1715
The model contained invalid expressions.
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/160

**Comment by modelica-trac-importer** [comment]
**Comment by christoph&#46;clauss on 14 Apr 2009 13:35 UTC**
Errors fixed on the SVN server.

Regards!

Christoph Clauss

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 14 Apr 2009 13:47 UTC**
Confirmed, fixed in 2949b333221713a5a5847cef9a280369e95f5a70. 

Christoph, for the future: Try to include the ticket number in the subversion commit message so it can be "back-linked" to the ticket. (E.g., "Fixes #160  at Electrical.Analog.Ideal (unknown variables)") :-)
